### PR TITLE
Mod3 Intermission Work

### LINKED ISSUE #212: Intermission Work between Mod2 & Mod3

### PR DESCRIPTION
**Pull request by memcmahon**
Closes #212

### CODE REVIEW

File: module3/intermission/review.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+title: Oregon Trail Mini Project
+---
+
+Let's build a small application to practice what we've learned in Mod2. Read throught the entire spec below before starting!
+
+## The Oregon Trail
```
Comment (zoefarrell) [inline module3/intermission/review.md:8]: Just calling out that without a starter repo we won't know if any students started this recommended project or how far they got. I think that's fine. What if we do a reflection at the end of week 1 about what they worked on over intermission and how it impacted their first week back?

File: module3/intermission/index.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+layout: page
+title: Intermission Work
+---
+
+Before staring Module 3, spend a minimum of 5 hours working through the following activities.
+
+### Required
+[Codeacademy Learn HTML - Elements and Structure](https://www.codecademy.com/learn/learn-html)
```
Comment (zoefarrell) [inline module3/intermission/index.md:9]: Agree with making this required! Trying to think what we could add as a deliverable here for accountability. Maybe they send us before day 1 the furthest exercise they got to in HTML and CSS and what their favorite exercise was?

**Review by zoefarrell — APPROVED** [review]

File: module3/intermission/review.md (hunk 1 above)
Comment (memcmahon) [reply module3/intermission/review.md:8]: In some ways I don't want to know.... like... I can build a world in my head where everyone is working on this 😆 

But its a good call-out.  Maybe I'll add something to drop us a like to your repo if you want feedback?

File: module3/intermission/index.md (hunk 2 above)
Comment (memcmahon) [reply module3/intermission/index.md:9]: Yea... maybe I'll go through the first section today, and try to find a good screenshot or something that shows that they have completed this 🧐

**Pull request merged**